src/main/java/phrasecount/PhraseExporter.java

```diff
@@ -0,0 +1,25 @@
+package phrasecount;
+
+import java.util.Collection;
+import java.util.Collections;
+
+import org.apache.accumulo.core.data.Mutation;
+import org.apache.fluo.recipes.accumulo.export.AccumuloExporter;
+import org.apache.fluo.recipes.core.export.SequencedExport;
+import phrasecount.pojos.Counts;
+import phrasecount.query.PhraseCountTable;
+
+/**
+ * Export code that converts {@link Counts} objects from the export queue to Mutations that are
+ * written to Accumulo.
+ */
+public class PhraseExporter extends AccumuloExporter<String, Counts> {
```

Comment:
It seems the readme already referenced this file, even though it didn't exist before.   I support it used to exist and we never updated the readme.  Now the readme is correct again.